tools/tgs4_scripts/PreCompile.bat

```diff
@@ -0,0 +1,2 @@
+@set TG_BUILD_TGS_MODE=1
+@.\Build.bat
```

Comment:
```
[12:18:11] dbg: Tgstation.Server.Host.System.ProcessExecutor (Instance:3|Job:40|Request:PUT /DreamMaker|User:1|Monitor:|Bridge:|Chat:|IR:)
    Launching process in S:\TGS4_INSTANCES\TGSTATION\Configuration\EventScripts: S:\TGS4_INSTANCES\TGSTATION\Configuration\EventScripts\PreCompile.bat S:\TGS4_INSTANCES\TGSTATION\Game\18babf41-bd45-4716-86ce-815267b963ef https://github.com/tgstation/tgstation
...
Tgstation.Server.Host.Jobs.JobException: Script PreCompile.bat exited with code 1:
'.\Build.bat' is not recognized as an internal or external command,
```

CWD is the event scripts directory, not the repository root which is passed in as the first parameter